streams/src/test/java/org/apache/kafka/streams/processor/internals/GlobalStreamThreadTest.java

```diff
@@ -163,7 +162,7 @@ public void shouldBeRunningAfterSuccessfulStart() {
     }
 
     @Test(timeout = 30000)
-    public void shouldStopRunningWhenClosedByUser() throws InterruptedException {
+    public void shouldStopRunningWhenClosedByUser() throws Exception {
```

Comment:
Can you explain why we should relax this declaration?

Reply:
Because it's a test. Nobody is expected to catch it, thus type information is unnecesary. It's a code style thing. No reason to declare any checked exceptions because if one is thrown the test fails anyway. Also avoid updates if the exception type changes.

Reply:
Let me know if I can merge. Guozhang gave a +1.

Reply:
Okeydoke, Sounds good to me.